Packs/Cybersixgill-ActionableAlerts/Classifiers/classifier-classifier-mapper-incoming-Cybersixgill-Actionable-Alerts.json

```diff
@@ -0,0 +1,31 @@
+{
+	"description": "",
+	"feed": false,
+	"id": "classifier-mapper-incoming-Cybersixgill-Actionable-Alerts",
```

Comment:
Might be better to change the `id`, `name` properties of the classifier to something friendlier (e.g. Cybersixgill Actionable Alerts Incoming Mapper) and update the defaultmapper in the yml file here if you do.

Reply:
Done